eem-sparks/src/main/java/com/nice/eem/sparkathon/UploadFileToS3.java

```diff
@@ -0,0 +1,55 @@
+package com.nice.eem.sparkathon;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.auth.InstanceProfileCredentialsProvider;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+
+public class UploadFileToS3 {
+
+
+    //uploading file to the given s3 bucket using client created by the builder
+    public static void uploadFileToS3(AmazonS3 s3Client, String bucketName, String fileName, String filePath) {
+        s3Client.putObject(bucketName, fileName, filePath);
+        System.out.println("File uploaded successfully");
+    }
+
+    //downloading file from s3 bucket using client created by the builder
+    public static void downloadFileFromS3(AmazonS3 s3Client, String bucketName, String fileName, String filePath) {
+        s3Client.getObject(bucketName, fileName);
+        System.out.println("File downloaded successfully");
+    }
+
+    /**
+     *
+     * @param iscloudDeployment
+     * @return
+     */
+    public AmazonS3 buildS3Client(boolean iscloudDeployment) {
+        AWSCredentials awsCredentials;
+        AmazonS3 s3Client;
+        String awsRegion = "us-east-1";
+        if (iscloudDeployment) {
+
+            String accessKey = "xxxxxxxxxxxxxxxxxx";
+            String secretKey = "xxxxxxxxxxxxxxxxxx";
+            awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
+            s3Client = AmazonS3ClientBuilder
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

AWS Region is set using a `String`. To explicitly set a publicly available region, we recommend that you use the [Regions](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/index.html?com/amazonaws/regions/Regions.html) enum.

[Learn more](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html#region-selection-choose-region)

Similar issue at line number 46.